salt/modules/rh_service.py

```diff
@@ -470,22 +472,34 @@ def reload_(name):
 
 def status(name, sig=None):
     '''
-    Return the status for a service, returns a bool whether the service is
-    running.
+    Returns a bool indicating whether the service is running.
```

Comment:
To keep this in line with other Salt documentation, instead of referring to the return as a "bool", this should say "Returns True if the service is running, otherwise False."